client/transactions/list/index.tsx

```diff
@@ -555,6 +561,14 @@ export const TransactionsList = (
 				!! search ||
 				!! typeIs ||
 				!! typeIsNot ||
+				!! typeIsNot ||
```

Comment:
Possible duplication?

Reply:
Certainly, it's a duplication. It has been removed.